scripts/us_ca_chhs/healthcare_facility_bed_counts/preprocess_csv.py

```diff
@@ -0,0 +1,261 @@
+            'Count_HospitalBed_IntermediateCareNursingBed': None,
+            'Count_HospitalBed_CongregateLivingHealthFacilityBed': None,
+            'Count_HospitalBed_IntermediateCareBed': None,
+            'Count_HospitalBed_CoronaryCareBed': None,
+            'Count_HospitalBed_IntensiveCareBed': None,
+            'Count_HospitalBed_IntensiveCareNewbornNurseryBed': None,
+            'Count_HospitalBed_PerinatalBed': None,
+            'Count_HospitalBed_UnspecifiedGeneralAcuteCareBed': None,
+            'Count_HospitalBed_BurnBed': None,
+            'Count_HospitalBed_PediatricBed': None,
+            'Count_HospitalBed_RenalTransplantBed': None,
+            'Count_HospitalBed_RehabilitationBed': None,
+            'Count_HospitalBed_AcuteRespiratoryCareBed': None,
+            'Count_HospitalBed_AcutePsychiatricCareBed': None,
+            'Count_HospitalBed_ChemicalDependencyRecoveryBed': None,
+            'Count_HospitalBed_PediatricIntensiveCareUnitBed': None,
+            'Count_HospitalBed_LaborAndDeliveryBed': None,
+            'Count_HospitalBed_IntermediateCareDevelopmentallyDisabledBed': None,
+            'Count_HospitalBed_PsychiatricHealthBed': None,
+            'Count_HospitalBed_PediatricDayAndRespiteCareBed': None,
+            'Count_HospitalBed_DialysisStationsBed': None,
+            'Count_HospitalBed_CorrectionalTreatmentCenterBed': None
+        }
+        new_row[stat_var] = row_dict['BED_CAPACITY']
+        new_df = new_df.append(new_row, ignore_index=True)
+
+new_df.to_csv('CA_Licensed_Healthcare_Facility_Types_And_Counts.csv')
+
+# Output as a tmcf file.
+# Facility node, facility containedIn a county
+TEMPLATE_MCF_FAC_1 = """
+Node: E:CA_Licensed_Healthcare_Facility_Types_And_Counts->E0
+healthcareFacilityName: C:CA_Licensed_Healthcare_Facility_Types_And_Counts->CALicensedHealthcareFacilityName
+typeOf: schema:Hospital
+dcid: C:CA_Licensed_Healthcare_Facility_Types_And_Counts->CALicensedHealthcareFacilityBedFACID
+healthcareFacilityType: C:CA_Licensed_Healthcare_Facility_Types_And_Counts->CALicensedHealthcareFacilityType
+containedIn: C:CA_Licensed_Healthcare_Facility_Types_And_Counts->GeoId
+"""
+
+# observation node, observationAbout facility
+TEMPLATE_MCF_TEMPLATE_1 = """
+Node: E:CA_Licensed_Healthcare_Facility_Types_And_Counts->E{index}
+typeOf: StatVarObservation
+variableMeasured: {stat_var}
+observationDate: C:CA_Licensed_Healthcare_Facility_Types_And_Counts->Date
+observationAbout: E:CA_Licensed_Healthcare_Facility_Types_And_Counts->E0
+value: C:CA_Licensed_Healthcare_Facility_Types_And_Counts->{stat_var}
+"""
+
+with open('CA_Licensed_Healthcare_Facility_Types_And_Counts.tmcf', 'w', newline='') as f_out:
+    f_out.write(TEMPLATE_MCF_FAC_1)
+
+    stat_vars = new_columns[5:]
+    for i in range(len(stat_vars)):
+        f_out.write(TEMPLATE_MCF_TEMPLATE_1.format_map({'index': i + 1, 'stat_var': stat_vars[i]}))
+
+# Proprocess for dataset2 and output tmcf file.
```

Comment:
Is there a benefit for stuffing both datasets in one script? They seem more like 2 separate imports.